packages/cli-lib/lang/en.lyaml

```diff
@@ -578,6 +578,18 @@ en:
                 instructions: "To update CLI permissions for \"{{ accountName }}\":
                   \n- Go to {{ url }}, deactivate the existing personal access key, and create a new one that includes Sandboxes permissions.
                   \n- Update the CLI config for this account by running `hs auth` and entering the new key.\n"
+          delete:
+            describe: "Delete a sandbox account"
+            examples:
+              default: "Deletes the sandbox account named MySandboxAccount."
+            success:
```

Comment:
Looks like the success and error keys aren't being used. Is that intentional because they'll be used later?

Reply:
Yeah, I had a bunch of code that I deleted as placeholder but left the translations in for now. Hopefully we'll get around to completing this soon, but totally happy to remove if we don't want dead end strings lying around

Reply:
Makes sense. I'll leave the decision up to you. If you're confident that you'll update this soon-ish then I'm okay with keeping it.

Reply:
Yep! cms dev infra team just got back to me with some info on how best to start working on the auth part, will start on it next Monday